lib/static/components/controls/browser-list/index.styl

```diff
@@ -1,52 +1,35 @@
-.browserlist
-    label-bg-color = #3333cc1a
-    button-hover-bg-color = #99999950
-
-    .array
-        width 220px
-
-    .rct-controls
-        width 100%
-        height 25px
-        margin-bottom 10px
-        border none
-        border-radius 5px
-        cursor pointer
-        background-color label-bg-color
-        transition background-color 150ms linear
-
-        &:hover
-            background-color button-hover-bg-color;
-
-    .rct-node-parent
-        .rct-node-leaf
-            .rct-node-icon
-                display none
-
-    .rct-node-icon
-        color black
-        text-align center
+.g-popup
+    z-index 9999
+.browserlist__filter
+    width 100%
+    display inline-flex
+    justify-content center
+    padding 4px 4px
+
+.browserlist__row
+    width 100%
+    display flex
+    align-items center
+    gap 4px
+
+    .browserlist__row_content
+        flex 1 1 auto
+
+    .action-button
+        opacity 0
+        
+.browserlist__row:hover
+    .action-button
+        opacity 1
```

Comment:
It should be:
```styl
.g-select-list__option:hover
    .browserlist__row
        .action-button
            opacity 1
```
That way the "Only"/"Except" button will be shown while hovering the entire line, including the check mark.
Currently "Only"/"Except" is not shown while hovering the check mark